docker-app/qfieldcloud/settings.py

```diff
@@ -234,7 +234,9 @@
 
 # QFieldCloud variables
 AUTH_TOKEN_LENGTH = 100
-AUTH_TOKEN_EXPIRATION_HOURS = 24 * 30
+AUTH_TOKEN_EXPIRATION_HOURS = int(
+    os.environ.get("QFIELDCLOUD_AUTH_TOKEN_EXPIRATION_HOURS", 24 * 30)
+)
 
```

Comment:
Note this is not backwards compatible with the existing environments, because `QFIELDCLOUD_AUTH_TOKEN_EXPIRATION_HOURS` variable will be an empty string and the default value will not be taken. See #589 .

Reply:
:see_no_evil: I will fix this in the evening

Reply:
Done